lips/lip-1.mdx

```diff
@@ -492,84 +457,234 @@ Represents a national ID.
+The sender can initiate a payment on-chain in case of an abort with `no-kyc-needed`.
 
-# Recipient Signature
-Once the recipient side is comfortable that it has received appropriate information and is ready for the transaction to go on-chain, the recipient side will provide their signature in order to support dual attestation. The receiver VASP will sign with the receiver compliance private key. The following is a concrete example of how to generate the Travel Rule dual attestation signable and the recipient signature using Libra Python Client SDK:
+### RSEND -> READY (Step 7)
+
+The Sender issues `PaymentCommand` to update an existing payment.
+
+The Sender VASP examines the KYC information from the Receiver and is satisfied the payment can proceed.
+
+* `Sender.Status.Status`: `ready_for_settlement`
+
+The payment should be executed on-chain by the sender (or settled in any other way) following the success of the command.
+
+### RSEND -> SABORT (Step 9)
+
+The Sender issues `PaymentCommand` to update an existing payment.
+
+The sender VASP receives the Receiver KYC information and decides it cannot proceed with the payment. It issues an abort command:
+
+* `Sender.Status.Status`: `abort`
+* `Sender.Status.abort_code`: `rejected`.
+
+## Steps of the protocol: The soft-match states and flows
+
+A _soft-match_ occurs when a VASP checks provided KYC, but cannot disambiguate whether a party to the payment is potentially high-risk or sanctioned, just with the information provided. This may occur because the identifiers used (names, dates of birth, addresses) may partially match and are noisy. In such cases the VASP may require further information from the other VASP to process the payment, some of which may require manual intervention and interaction.
+
+The flows below allow the Receiver VASP and the Sender VASP to request additional KYC information from each other.
+
+### Receiver soft-match (SINIT -> RSOFT -> SSOFTSEND)
+
+After the Sender initiates a payment by providing KYC information (SINIT), the Receiver may determine they require more information to disambiguate a match. In that case they commit a command to set `Receiver.Status.Status` to `soft-match` (state RSOFT) (Step 5) or abort (step 9). The sender may respond with a command that populates the `Sender.additional_kyc_data` (Step 6), which sets the flag `Sender.additional_kyc_data` (state SSOFTSEND) or abort (SABORT, Step not shown in diagram). Finally, if the Receiver is satisfied with the additional information they move to provide all information necessary to go to RSEND (Step 4, see above. This includes receiver KYC information and recipient signature). Otherwise, the receiver can abort to state RABORT (Step 8).
```

Comment:
```suggestion
After the Sender initiates a payment by providing KYC information (SINIT), the Receiver may determine they require more information to disambiguate a match. In that case they commit a command to set `Receiver.Status.Status` to `soft-match` (state RSOFT) (Step 5) or abort (step 9). The sender may respond with a command that populates the `Sender.additional_kyc_data` (Step 6), which sets the field `Sender.additional_kyc_data` (state SSOFTSEND) or abort (SABORT, Step not shown in diagram). Finally, if the Receiver is satisfied with the additional information they move to provide all information necessary to go to RSEND (Step 4, see above. This includes receiver KYC information and recipient signature). Otherwise, the receiver can abort to state RABORT (Step 8).
```